flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/materializedtable/AlterMaterializedTableAsQueryOperation.java

```diff
@@ -28,13 +28,27 @@
 
 /** Operation to describe an ALTER MATERIALIZED TABLE AS query operation. */
 @Internal
-public class AlterMaterializedTableAsQueryOperation extends AlterMaterializedTableChangeOperation {
+public class AlterMaterializedTableAsQueryOperation extends AlterMaterializedTableOperation {
```

Comment:
why `Change` was dropped from name?

It was named in same pattern as for table and it works with `TableChange` objects`